src/components/PermissionsManagement/UserPermissionsPopUp.jsx

```diff
@@ -180,6 +180,7 @@ function UserPermissionsPopUp({ allUserProfiles, getAllUsers, roles, authUser, d
             rolePermissions={userPermissions}
             immutablePermissions={actualUserRolePermission}
             editable={!!actualUserProfile}
+            isEditing
```

Comment:
This reminded me... You need to differentiate between 'postRole', 'putRole', and 'putUserProfilePermissions', so you need extra information to distinguish role vs user permissions. Could either be a second boolean or could swap isEditing to something like useCase for 'newRole', 'editRole', or 'editUser'. Up to you. Sorry for not catching that earlier

Reply:
Will do asap. Thanks for letting me know!

Reply:
done